ros/interfaces.bzl

```diff
@@ -163,10 +164,11 @@ def _cc_ros_generator_aspect_impl(target, ctx):
         args = ctx.actions.args()
         args.add("-o", msg_header.dirname)
         args.add("-p", ros_package_name)
+        args.add("-e", templates[0].dirname)
         args.add_all(include_flags)
         args.add(src)
         ctx.actions.run(
-            inputs = all_srcs,
+            inputs = all_srcs.to_list() + templates,
```

Comment:
For performance reasons I'd create a depset from all_srcs and ctx.attr._templates.files (both transitive) and pass that to inputs.

Reply:
Is https://github.com/mvukov/rules_ros/pull/7/commits/2486e1db4ca258c7f889d5ce39efb418c3aae139 what you had in mind?

Reply:
Optimized a bit. Thanks for the contribution.